### PR TITLE
[TextFields] Changing insets to react to new property

### PR DESCRIPTION
**Pull request by willlarche**

### CODE REVIEW

File: components/TextFields/src/private/MDCTextInputCommonFundament.m [hunk 1]
```diff
@@ -665,7 +680,46 @@ - (void)setTextInput:(UIView<MDCTextInput> *)textInput {
 }
 
 - (UIEdgeInsets)textInsets {
-  return _textInput.textInsets;
+  UIEdgeInsets textInsets = UIEdgeInsetsZero;
+
+  textInsets.top = MDCTextInputFullPadding;
+
+  CGFloat leadingOffset = MDCCeil(self.leadingUnderlineLabel.font.lineHeight * 2.f) / 2.f;
```
Comment (ianegordon) [inline components/TextFields/src/private/MDCTextInputCommonFundament.m:687]: Consider adding a comment indicating why the leading and trailing offsets are linked to lineHeight.
I'm not caffeinated yet, but do we need to ceil(x * 2) / 2?  Can we not just ceil(x) ?

**Review by ianegordon — APPROVED** [review]
Please address comments before merge.

File: components/TextFields/src/private/MDCTextInputCommonFundament.m (hunk 1 above)
Comment (romoore) [reply components/TextFields/src/private/MDCTextInputCommonFundament.m:687]: Looks like rounding code that permits half-points?

Comment (willlarche) [reply components/TextFields/src/private/MDCTextInputCommonFundament.m:687]: Yes, @romoore . This is a quick estimation of uilabel height based on font line height. I measure a bunch and saw the pattern that it was ceiling to .5's. 

Would be fabulous if someone could find out for SURE what the formula is and put it in typography or something.

Comment (willlarche) [reply components/TextFields/src/private/MDCTextInputCommonFundament.m:687]: @ianegordon , these are the estimated measurements of the height that will be needed for the leading and trailing underline labels. So it's not leading or trailing space. It's leading and trailing underline labels' heights.

Comment (willlarche) [reply components/TextFields/src/private/MDCTextInputCommonFundament.m:687]: Improving estimation in a new PR because conflicts.

**Pull request merged**